XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeExtrasPage.resx

```diff
@@ -382,6 +382,6 @@
     <value>RollingUpgradeExtrasPage</value>

```

Comment:
When I changed `[XenCenter_No_Space]Main` to `[XenCenter_No_Space]`, SelectMultipleVMDestinationPage.resx had `XenCenterMain`, hence was not changed. When I raised the current PR though, I rebased on master where SelectMultipleVMDestinationPage.resx had changed in the mean time to `[XenCenter_No_Space]Main`. I'll change this particular instance to be consistent. Note there seem to exist many instances of `XenCenterMain` left there since the very first time we replaced the brand names with placeholders, but they don't seem to cause any issues.